### ISSUE TITLE
condo creates tag before PR is merged from hotfix/* -> master

### ISSUE THREAD

**Issue by brandon-a-hay** [automotiveMastermind/condo#195]
When creating a pr from a hotfix branch condo created the 4.11.0 tag. It should only happen once the branch is merged into master.
PR: https://automotivemastermind.visualstudio.com/AutomotiveMastermind/AutomotiveMastermind%20Team/_git/BusinessLogic/pullrequest/5311?_a=overview

**Comment by dmccaffery** [comment]
@brandon-a-hay is this still an issue?
